app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
```

Comment:
- [ ] Great job working on the wireframe but I think it needs more improvement to be like the given wireframe since the logging button should have the same width as the inputs so kindly update it, ✅💯⭐ and please make the signup page the same as well  ♻



![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/5e30b012-bb5c-4409-b0e8-50d486ec9156)

- Your
![image](https://github.com/theresetuyi/Blog-app/assets/58553711/6a4af24a-9456-47f4-818e-9caabb0410d0)


- Given
![image](https://github.com/ShaliniNayan/Blog-app/assets/58553711/be212d76-1bf7-492e-8396-f7438b8ea7b9)